### ISSUE TITLE
Additional Placeholders for Skills integration ease of use

### ISSUE THREAD

**Issue by Jhezreel708** [mmopluginteam/mmocore#129]
> **Is your feature request related to a problem? Please describe.**]

lessen the headache of targeting mechanics for mythic mobs damaging party members and to add more customization for skills. 
like if Fireball is level one it will be a small particle only on level 10 fire ball it will have additional effects thus we need the skill level placeholder for skills.

> **Describe the solution you'd like**
> A clear and concise description of what you want to happen.
> Add placeholders this would help much since most of the skills can be made on mythicmobs and there are lots of work >arounds that can be implemented

```
placeholders:
%mmocore.guild.name% <- this can also be used for the skills.

%mmocore.party.owner% <- easy to use instead of having a complex party system just compare the party leader name or UUID or tag.
%mmocore.skill.(skillname).level% <- gives additional creativity making some skills more alive.

```

**Examples of use:**
%mmocore.skill.FireBolt.level%

if the **skill level** is on level 1
it will only do a small firebolt skill
if the skill is on **level** 10
it will do a large firebolt skill using cast instead mythicmobs mechanic.

%mmocore.party.owner%
Group Heal/ Area heal skill.

upon casting a group heal skill it would heal all the players around you within 10 blocks that's also inside your party.
-exampleskill ~PIR{r=10}
`target conditions: %mmocore.party.owner% (%mmocore.guild.name%) < is the same. ( ) <- is optional.`

this would help a lot of healing skills dedicated for party.

Aalright after all that comes down to the question of `how about for players who just want to /heal/ all players even tho it's not within the party`
basically same option for the skill casting of Shift <- [Heal party/guild members] only or [standing = heal all players]
plenty of alternatives for it;
